.github/workflows/docs/frontend/frontend_npm_release.md

```diff
@@ -37,6 +37,7 @@ jobs:
     uses: jupiterone/.github/.github/workflows/frontend_npm_release.yml
     secrets:
       NPM_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}
+      AUTO_GITHUB_PAT_TOKEN: ${{ secrets.AUTO_GITHUB_PAT_TOKEN }} 
```

Comment:
These changes seem unrelated?

Reply:
They are, just missed a prop on the documentation